src/tennisSimulator/Player.java

```diff
@@ -50,6 +46,7 @@ public boolean isServing() {
 
     public void setServing(boolean serving) {
         isServing = serving;
+        System.out.println(name + " is serving");
```

Comment:
When setServing will be called with false value this method will still print that this player is serving